packages/nextjs-mf/src/loaders/fixImageLoader.ts

```diff
@@ -1,25 +1,22 @@
 import type { LoaderContext } from 'webpack';
-
 import { Template } from 'webpack';
 import path from 'path';
 
 /**
- * This loader was specially created for tunning next-image-loader result
- *   see https://github.com/vercel/next.js/blob/canary/packages/next/build/webpack/loaders/next-image-loader.js
- * It takes regular string
- *   `export default {"src":"/_next/static/media/ssl.e3019f0e.svg","height":20,"width":20};`
- * And injects the following code:
- * for ssr:
- * remote scope of specific remote url
- * for csr:
- * document.currentScript.src
- * after that, it will choose the full uri before _next
- *
- * for example:
+ * This loader is specifically created for tuning the next-image-loader result.
+ * It modifies the regular string output of the next-image-loader.
+ * For server-side rendering (SSR), it injects the remote scope of a specific remote URL.
+ * For client-side rendering (CSR), it injects the document.currentScript.src.
+ * After these injections, it selects the full URI before _next.
+ * 
+ * @example
```

Comment:
The logic inside the 'if' statement can be simplified by using a return early pattern. This can improve readability and maintainability.